synapse/lib/cell.py

```diff
@@ -1155,6 +1155,9 @@ async def __anit__(self, dirn, conf=None, readonly=False, parent=None):
         self.usermetadb = self.slab.initdb('user:meta')  # useriden + <valu> -> dict valu
         self.rolemetadb = self.slab.initdb('role:meta')  # roleiden + <valu> -> dict valu
 
+        # for runtime cell configuration values
+        self.slab.initdb('cell:conf')
```

Comment:
```suggestion
        self.confdb = self.slab.initdb('cell:conf')
```
Or something? I realize the return value is just a string but this has been the standard method of using the slab db names in every other place. Are you wanting to change how we do this?

Reply:
The more I think about it, using (something like) `self.confdb` makes more sense when the slab is created on the `Cell` because it's more likely to get out of sync on subclasses.